js/main.js

```diff
@@ -1,9 +1,18 @@
 // main.js
 // Contains Javascript functions.....
 
-// Easter egg
-var rawr = function() {
-  document.getElementsByClassName('container')[0].style.paddingTop = 0;
-  document.getElementsByClassName('banner')[0].style.display = "none";
-  document.getElementsByClassName('soon')[0].innerHTML = "<img height='200px' src='images/rawrburt.png'>";
-}
+"use strict";
+
+(function() {
+  var soon = document.getElementsByClassName('soon')[0];
+
+  // Easter egg
+  document.getElementById("rawr").onclick = function() {
+      soon.innerHTML = "";
+      document.getElementsByClassName('banner')[0].style.display = "none";
+      document.getElementsByClassName('container')[0].style.paddingTop = 0;
+      var rawrburt = document.createElement("img")
+      rawrburt.src = "images/rawrburt.png";
+      soon.appendChild(rawrburt);

```

Comment:
`<h1>` elements are usually reserved for headlines
### special text ###
but they are technically containers so this sort of works (not the way I would do things, though)